user-operator/src/main/java/io/strimzi/operator/user/operator/ScramShaCredentials.java

```diff
@@ -23,7 +22,7 @@
  * Utility class for managing Scram credentials
  */
 public class ScramShaCredentials {
-    private static final Logger log = LogManager.getLogger(ScramShaCredentials.class.getName());
+    private static final ReconciliationLogger LOGGER = ReconciliationLogger.create(ScramShaCredentials.class.getName());
```

Comment:
Same as above ... this looks like it doesn't need `ReconciliationLogger` right now. But I think it should be really using it since it is part of a reconciliation.